code/modules/client/preferences_spawnpoints.dm

```diff
@@ -70,21 +70,11 @@ GLOBAL_VAR(spawntypes)
 	if(!istype(victim))
 		return
 	var/area/A = get_area(victim)
-	var/list/spots = list()
-
 	for(var/obj/machinery/cryopod/C in A)
-		if(!C.occupant)
```

Comment:
Removing this check could result in people getting spawned from the same cryopod.

Reply:
![waterfox_9yYGsLNCiq](https://user-images.githubusercontent.com/54416597/90313474-8d878e80-df15-11ea-8fd7-35ca93725b31.png)
it's there, but git sucks DICK